### PR TITLE
Updated the Linux kernel info on the software page

### LINKED ISSUE #7391: Linux kernel info out of date on Software recommendations page
# General issue

## Description:

Regarding https://devdocs.magento.com/guides/v2.3/performance-best-practices/software.html

Operating System section specifically

>  The Linux kernel supports a mechanism to “reuse” and “recycle” TCP connections. Be aware that more aggressive recycling than re-use may cause issues on the load balancers. To enable these kernel settings, set the following values in /etc/sysctl.conf:
net.ipv4.tcp_tw_recycle = 1
net.ipv4.tcp_tw_reuse = 1

## Possible solutions:

Well first and foremost net.ipv4.tcp_tw_recycle doesnt exist past kernel 4.12
see [https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=4396e46187ca5070219b81773c4e65088dac50cc](url)
also [https://stackoverflow.com/questions/6426253/tcp-tw-reuse-vs-tcp-tw-recycle-which-to-use-or-both](url)
not even notated anymore in sysctl documention [https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt](url)

and net.ipv4.tcp_tw_reuse is useless for incoming connections

> Enabling net.ipv4.tcp_tw_reuse is useless for incoming connections.

please see [https://vincent.bernat.ch/en/blog/2014-tcp-time-wait-state-linux](url)

### PR DESCRIPTION
**Pull request by chiranjeevi-cj**
## Purpose of this pull request

This pull request (PR) Updates the Linux kernel info on the software page

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/performance-best-practices/software.html
-  https://devdocs.magento.com/guides/v2.4/performance-best-practices/software.html

Fixes #7391

### CODE REVIEW

File: src/guides/v2.3/performance-best-practices/software.md [hunk 1]
```diff
@@ -24,10 +24,12 @@ See [Magento 2.2.x technology stack requirements]({{page.baseurl}}/install-gde/s
 
 ## Operating system
 
-Operating system configurations and optimizations are similar for Magento as other high-load web applications. As the number of concurrent connections handled by the server increases, the number of available sockets can become fully allocated. The Linux kernel supports a mechanism to "reuse" and "recycle" TCP connections. Be aware that more aggressive recycling than re-use may cause issues on the load balancers. To enable these kernel settings, set the following values in `/etc/sysctl.conf`:
+Operating system configurations and optimizations are similar for Magento as other high-load web applications. As the number of concurrent connections handled by the server increases, the number of available sockets can become fully allocated. The Linux kernel supports a mechanism to "reuse" TCP connections. To enable these kernel settings, set the following values in `/etc/sysctl.conf`:
```
Comment (BarnyShergold) [inline src/guides/v2.3/performance-best-practices/software.md:27]:
```suggestion
Operating system configurations and optimizations are similar for Magento as other high-load web applications. As the number of concurrent connections handled by the server increases, the number of available sockets can become fully allocated. The Linux kernel supports a mechanism to "reuse" TCP connections. To enable this mechanism, set the following value in `/etc/sysctl.conf`:
```

File: src/guides/v2.3/performance-best-practices/software.md [hunk 2]
```diff
@@ -24,10 +24,12 @@ See [Magento 2.2.x technology stack requirements]({{page.baseurl}}/install-gde/s
...
+
+{:.bs-callout-info}
+Enabling net.ipv4.tcp_tw_reuse is useless for incoming connections.
```
Comment (BarnyShergold) [inline src/guides/v2.3/performance-best-practices/software.md:30]:
```suggestion
Enabling net.ipv4.tcp_tw_reuse has no effect on incoming connections.
```

File: src/guides/v2.4/performance-best-practices/software.md [hunk 3]
```diff
@@ -24,10 +24,12 @@ See [Magento technology stack requirements]({{page.baseurl}}/install-gde/system-
 
 ## Operating system
 
-Operating system configurations and optimizations are similar for Magento as other high-load web applications. As the number of concurrent connections handled by the server increases, the number of available sockets can become fully allocated. The Linux kernel supports a mechanism to "reuse" and "recycle" TCP connections. Be aware that more aggressive recycling than re-use may cause issues on the load balancers. To enable these kernel settings, set the following values in `/etc/sysctl.conf`:
+Operating system configurations and optimizations are similar for Magento as other high-load web applications. As the number of concurrent connections handled by the server increases, the number of available sockets can become fully allocated. The Linux kernel supports a mechanism to "reuse" TCP connections. To enable these kernel settings, set the following values in `/etc/sysctl.conf`:
```
Comment (BarnyShergold) [inline src/guides/v2.4/performance-best-practices/software.md:27]:
```suggestion
Operating system configurations and optimizations are similar for Magento as other high-load web applications. As the number of concurrent connections handled by the server increases, the number of available sockets can become fully allocated. The Linux kernel supports a mechanism to "reuse" TCP connections. To enable this mechanism, set the following value in `/etc/sysctl.conf`:
```

File: src/guides/v2.4/performance-best-practices/software.md [hunk 4]
```diff
@@ -24,10 +24,12 @@ See [Magento technology stack requirements]({{page.baseurl}}/install-gde/system-
...
+
+{:.bs-callout-info}
+Enabling net.ipv4.tcp_tw_reuse is useless for incoming connections.
```
Comment (BarnyShergold) [inline src/guides/v2.4/performance-best-practices/software.md:30]:
```suggestion
Enabling net.ipv4.tcp_tw_reuse has no effect on incoming connections.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few phrasing suggestions

**Review by BarnyShergold — APPROVED** [review]
Thanks for your work!

**Review by jeff-matthews — APPROVED** [review]

**Pull request merged**